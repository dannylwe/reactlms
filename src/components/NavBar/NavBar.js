import React, { Component } from "react";

class NavBar extends Component {
  render() {
    return (
      <>
        <nav className="navbar navbar-dark bg-dark">
          <a className="navbar-brand" href="/dashboard">
            Home
          </a>
          <a
            className="navbar-brand"
            href="/login"
            onClick={() => {
              localStorage.clear();
            }}
          >
            Logout
          </a>
        </nav>
      </>
    );
  }
}

export default NavBar;
