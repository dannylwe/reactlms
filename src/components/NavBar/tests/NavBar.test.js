import React from "react";
import renderer from "react-test-renderer";
import NavBar from "../NavBar";

describe("NavBar", () => {
  it("should render without failing", () => {
    const wrapper = renderer.create(<NavBar />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
