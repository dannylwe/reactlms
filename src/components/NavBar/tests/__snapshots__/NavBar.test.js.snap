// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should render without failing 1`] = `
<nav
  className="navbar navbar-dark bg-dark"
>
  <a
    className="navbar-brand"
    href="/dashboard"
  >
    Home
  </a>
  <a
    className="navbar-brand"
    href="/login"
    onClick={[Function]}
  >
    Logout
  </a>
</nav>
`;
